packages/graphiql-plugin-react-doc-explorer/.eslintrc

```
@@ -0,0 +1,22 @@
+{
+  "extends": [
```

```suggestion
  "root": true,
  "extends": [
```
this config do nothing, without `root: true` always root eslint config is taken in priority